src/Hodor/JobQueue/JobOptions/Validator.php

```diff
@@ -85,4 +86,19 @@ private function validateRunAfter(array $options)
 
         throw new Exception('\'run_after\' must be an instance of \DateTime');
     }
+
+    /**
+     * @param array $options
+     * @throws Exception
+     */
+    private function validateJobRank(array $options)
+    {
+        if (is_int($options['job_rank'])
```

Comment:
Correct. The `validateJobOptions` method loops through each of the job options passed to the job and calls the `validateJobRank` method only if `job_rank` is in the job options.